NetSDK/Services/Client/Classes/Client.cs

```diff
@@ -40,5 +46,19 @@ public string GetTreatment(string key, string feature, Dictionary<string, object
                 return Control;
             }
         }
+
+        private long CurrentTimeMillis()
+        {
+            var Jan1st1970 = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Why do we need this? You are just measuring a delta. No need to make this calculation I don't think.